src/asm3/configuration.py

```diff
@@ -354,7 +354,8 @@
     "WeightChangeLog": "Yes",
     "WeightChangeLogType": "4",
     "CardcomSuccessURL": "https://secure.cardcom.solutions/DealWasSuccessful.aspx",
-    "CardcomErrorURL": "https://secure.cardcom.solutions/DealWasUnSuccessful.aspx"
+    "CardcomErrorURL": "https://secure.cardcom.solutions/DealWasUnSuccessful.aspx",
+    "CardcomDocumentType": "3"
```

Comment:
Unless the Cardcom URL changes per customer, these shoud be part of sitedefs.py. CardcomDocumentType looks like a constant that should be in PaymentProcessor subclass?

Reply:
They're actually per customer - they're what Cardcom will redirect to on success or error. CardcomDocumentType is a default value that can be changed in the config. 3=receipt which may be used by pretty much everybody (that's why I set it as default) but other values can be set depending on the org's entity type (nonprofit/forprofit) and the whether they want Cardcom to produce a document or not

Reply:
ah, disregard, that's fine then. Same for the others, I'll do the merge and we can tweak from there. Thanks!